rpcapd/rpcapd.c

```diff
@@ -91,30 +91,34 @@ static void main_cleanup_childs(int sign);
 */
 static void printusage(void)
 {
+	(void)fprintf(stderr, "RPCAPD, a remote packet capture daemon.\n"
+	"Compiled with %s\n\n", pcap_lib_version());
 	char *usagetext =
-	"USAGE:\n"
+	"USAGE:"
 	" "  PROGRAM_NAME " [-b <address>] [-p <port>] [-6] [-l <host_list>] [-a <host,port>]\n"
```

Comment:
Not to self: there is no -6 option. Or is there?

Reply:
No, there's a `-4` option.  Fixed in 8f26928e66da7072957b0c3c309f4f89005aec3b.

Reply:
👍